units.libs.lua

```diff
@@ -183,7 +183,7 @@ StaticLibrary {
 		Filters = {
 			{ Pattern = "[/\\]windows[/\\]"; Config = { "win32-*", "win64-*" } },
 			{ Pattern = "[/\\]macosx[/\\]"; Config = "mac*-*" },
-			{ Pattern = "[/\\]x11[/\\]"; Config = { "x11-*" } },
+			{ Pattern = "[/\\]X11[/\\]"; Config = { "x11-*" } },
```

Comment:
Very strange! I wonder what the differences in our git environments are. I'll try to open a PR to fix, but I may not have a chance to for a couple of days, so you may need to fix it up if you need it before then sorry!